source/index.js

```diff
@@ -13,6 +13,7 @@ renderer.roundPixels = true
 
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
+game.playerCount = 0
 game.addChild(new Junkership())
```

Comment:
Thought... we could pull collisionCheck from Utility and put it in the GameContainer class. Makes sense that it's the "game" that's checking for collisions. Looking within itself, as it were. Cuts down on the number of files.

Opinions?